app/services/idv/flows/in_person_flow.rb

```diff
@@ -53,6 +53,14 @@ def initialize(controller, session, name)
       def self.session_idv(session)
         session[:idv] ||= {}
       end
+
+      def extra_analytics_properties
+        return {} if @flow_session[:pii_from_user][:same_address_as_id].nil?
+        {
+          same_address_as_id: @flow_session[:pii_from_user][:same_address_as_id] == 'true' ? 1 : 0,
```

Comment:
If we support a zero value in analytics for `same_address_as_id`, then I recommend testing for it.

Reply:
it sounds like a boolean property to me. I know that CloudWatch turns booleans into `1`/`0` but I think logically it's better to write the value as a boolean

Reply:
It is a boolean, so @zachmargolis it sounds like I probably don't need to do the coercion to integers at all?

Reply:
Based on the check here, it appears to be a string representing a boolean. Coercion may still be necessary, but to boolean instead of string.

Reply:
yeah if we are worried it's either a string or a boolean, this will convert to boolean:

```suggestion
          same_address_as_id: @flow_session[:pii_from_user][:same_address_as_id].to_s == 'true',
```

Reply:
Ok, since cloudwatch converts booleans, I'll go ahead and make this into a boolean value.

Reply:
> If we support a zero value in analytics for `same_address_as_id`, then I recommend testing for it.

@NavaTim I just added a potentially questionable solution to help with this. I was trying to not replicate a bunch of code in the analytics_spec and associated helpers, but let me know if this is a bad idea.